.github/workflows/execute_tutorial_notebooks_in_external_repo.yml

```diff
@@ -0,0 +1,88 @@
+########################################################################################
+# Trigger the notebooks' execution in the external repository.
+#
+# If the "main" branch triggered this event, then the nb execution is triggered in the
+# "main" branch, otherwise in the dev branch.
+#
+# Naming convention used in this workflow:
+#
+# "main repository": Project repository with the python package and the documentation.
+# "linked repository": Repository where the example gallery of jupyter notebooks is
+#                      stored.
+########################################################################################
+name: Trigger notebook execution in the external repository
+
+env:
+  # URL for the external repository linked with the notebooks in this project.
+  LINKED_REPO: pysteps/pysteps_tutorials
```

Comment:
I'm just being annoying here, but for other pysteps-related repos we used the hyphen, so what about changing the name to `pysteps-tutorials`?